shared/rakefiles/secrets.rb

```diff
@@ -67,8 +69,19 @@ def self.collect_secrets()
       end
     end
 
-    encryption_keys.uniq!
-    ENV["TF_VAR_encryption_keys"] = %Q|[ #{encryption_keys.join(", ")} ]|
+    encryption_keys = {}
+    secrets_config = YAML.load(File.read("./modules/gcp-secret-mgmt/config.yaml"))
+    secrets_config["encryption_keys"].each do |encryption_key|
+      encryption_keys[encryption_key] = %Q|"#{encryption_key}"|
+    end
+
+    unless (collected_secrets.keys - encryption_keys.keys).empty?
```

Comment:
I would do `leftover_keys = collected_secrets.keys - encryption_keys.keys` and then use `leftover_keys` instead of performing this calculation twice.

Reply:
Sounds good!